pkg/storagepool/storagepool_template_engine.go

```diff
@@ -14,16 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// This file has storagepool specific implementation of cas template engine
+// Package storagepool provides a specific implementation of CAS template engine
 package storagepool
 
 import (
 	"fmt"
+	"strings"
+
```

Comment:
Thanks for the PR.
Can we remove this extra blank line.
@benkim0414

Reply:
It was because of my vim-go settings.
I fixed it.

Cheers.